server/chat-plugins/scavengers.js

```diff
@@ -193,7 +193,7 @@ function formatQueue(queue = [], viewer, room, broadcasting) {
 			const queuedBy = item.hosts.every(h => h.id !== item.staffHostId) ? ` / ${item.staffHostId}` : '';
 			let questions;
 			if (!broadcasting && (item.hosts.some(h => h.id === viewer.id) || viewer.id === item.staffHostId)) {
-				questions = item.questions.map((q, i) => i % 2 ? `<span style="color: green"><em>[${Chat.escapeHTML(q.join(' / '))}]</em></span><br />` : Chat.escapeHTML(q)).join(" ");
+				questions = item.questions.map((q, i) => i % 2 ? Chat.html`<span style="color: green"><em>[${q.join(' / ')}]</em></span><br />` : Chat.escapeHTML(q)).join(" ");
```

Comment:
```suggestion
				questions = item.questions.map((q, i) => i % 2 ? (
					Chat.html`<span style="color: green"><em>[${q.join(' / ')}]</em></span><br />`
				) : (
					Chat.escapeHTML(q)
				)).join(" ");
```

This is pretty unreadable without this.

Reply:
```
   198:1  error  Expected indentation of 5 tabs but found 4  indent
   198:6  error  Bad line breaking before and after ':'      operator-linebreak
```

Reply:
Can you play around with it until it stops complaining? Maybe it needs parens?

Reply:
```suggestion
				questions = item.questions.map((q, i) => (i % 2 ?
					Chat.html`<span style="color: green"><em>[${q.join(' / ')}]</em></span><br />`
				:
					Chat.escapeHTML(q)
				)).join(" ");
```

Try this?

Reply:
I looked into this and I don't think `eslint` allows a bare `:` at all; you either need to have it all on one line (if it fits), or you can (as currently configured in `.eslintrc`) only allow a newline after the `:` to break the line up, or the other setting would only allow a newline before the `:` to break the line up. (Unless you override `eslint` for this line of course.)

Reply:
Can you override `eslint` to permit it? Feel free to do that in a separate PR; I won't block this one on that.